### PR TITLE
Pokemon Go: Created cheat sheet

### PR DESCRIPTION
**Pull request by tagawa**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [X] New Instant Answer
    - [X] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

A new cheat sheet explaining the features of Pokemon Go.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

@zekiel @edgesince84 As the resident experts, could you check this looks OK please?

---

Instant Answer Page: https://duck.co/ia/view/pokemon_go_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @tagawa

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pokemon go cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/pokemon_go_cheat_sheet_mobile.png?7ZQKWLTJPK)

File: share/goodie/cheat_sheets/json/pokemon-go.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
+{
+    "id": "pokemon_go_cheat_sheet",
+    "name": "Pokémon GO",
+    "description": "A summary of some items and terminology used in the Pokémon GO game",
```
Comment (VeerpalBrar) [inline share/goodie/cheat_sheets/json/pokemon-go.json]: Remove "some" to be more concise. So " A summary of items and terminology used in the Pokemon Go app."

File: share/goodie/cheat_sheets/json/pokemon-go.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                "key": "Battle",
+                "val": "A battle at a friendly gym gives you experience points, needed for levelling up. A battle at a rival gym is a chance to take over and claim that gym. During a battle you can attack and dodge your opponent"
+            },
+            {
+                "key": "Gym",
+                "val": "Indicated as a large tower on your map, these are the only places where real battles can take place. You need to reach at least level 5 to use a gym"
+            },
+            {
+                "key": "CP",
+                "val": "CP, or combat power, gives your Pokemon strength for battles. CP can be increased by using candy or stardust to level-up or evolve your Pokemon"
```
Comment (VeerpalBrar) [inline share/goodie/cheat_sheets/json/pokemon-go.json]: Use Pokémon rather then pokemon to me more correct.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pokemon go cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/pokemon_go_cheat_sheet_mobile.png?yiCnrRgLkQ)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=pokemon go cheatsheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/pokemon_go_cheat_sheet_mobile.png?d9BzUN0lzW)

**Comment by tagawa** [comment]
Thanks very much for the good suggestions @VeerpalBrar - I've updated the cheat sheet.

**Comment by tagawa** [comment]
No further feedback so I'm going to merge this. Thanks again for the review @VeerpalBrar

**Pull request merged**